src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java

```diff
@@ -317,7 +325,27 @@ protected void createAnnotationEngine() {
     // get VariantContexts from input gVCFs and regenotype
     @Override
     public void apply(VariantContext variant, ReadsContext reads, ReferenceContext ref, FeatureContext features) {
-        regenotypeVC(variant);
+        if (!variant.hasAllele(Allele.NON_REF_ALLELE)) {
+            throw new GATKException("Variant Context at " + variant.getContig() + ":" + variant.getStart() + " does not contain a <NON-REF> allele. This tool is only intended for use with GVCFs.");
+        }
+        VariantContext newVC = annotationsToRemove.size() > 0 ? vcFormatAnnotationsRemoved(variant) : variant;
+        regenotypeVC(newVC);
+    }
+
+    /**
+     * Remove format level annotations from genotype in variant context.
+     *
+     * @param vc variant context to remove format annotations from
+     * @return variant context with format annotations removed from genotype
+     */
+    private VariantContext vcFormatAnnotationsRemoved(final VariantContext vc) {
```

Comment:
The grammar of this method name feels weird.  Like usually they're verb phrases. I would prefer "removeVCFormatAnnotations"